src/test/java/com/synopsys/integration/blackduck/comprehensive/NotificationsTestIT.java

```diff
@@ -68,8 +75,23 @@ public void testProjectNotifications() throws IntegrationException, InterruptedE
         ProjectVersionWrapper projectVersionWrapper2 = projectService.syncProjectAndVersion(projectSyncModel2, true);
 
         // DELETE
-        blackDuckApiClient.delete(projectVersionWrapper2.getProjectVersionView());
-        blackDuckApiClient.delete(projectVersionWrapper.getProjectView());
+        blackDuckApiClient.delete(projectVersionWrapper2.getProjectVersionView()); // TOME looks to be successful, v#2 on BD
+        blackDuckApiClient.delete(projectVersionWrapper.getProjectView()); // TOME deleting #1, fails intermittently and leaves project version behind when test case exits
+        // TODO option 1: sleep and/or retry
+        // TODO option 2: just retry on jenkins .. though sometimes it very consistently fails ... ?
+        // TODO change the notifications to something else since this test is really about just checking the correct notifications are sent?
+            // other possible notifications:
+        /**
+         *     BOM_EDIT,
+         *     LICENSE_LIMIT,
+         *     POLICY_OVERRIDE,
+         *     PROJECT,
+         *     PROJECT_VERSION,
+         *     RULE_VIOLATION,
+         *     RULE_VIOLATION_CLEARED,
+         *     VERSION_BOM_CODE_LOCATION_BOM_COMPUTED,
+         *     VULNERABILITY;
+         */
```

Comment:
@DanaMaxfield I was going to try the 3rd TODO comment and maybe see if I can get the pipeline green again? Seems more of a solid solution than option 1